server/cluster.go

```diff
@@ -244,6 +246,8 @@ type ProxyResponse struct {
 	SkipSid string
 	// User id of the affected user.
 	Uid types.Uid
+	// Response params.
+	Params map[string]interface{}
```

Comment:
Why now use something like this?
```go
type ProxyResponseParams struct {
   background bool
} 
```

Reply:
Or a pointer to it.

Reply:
Or even just add `IsBackground bool` to `ProxyResponse` struct?

Reply:
Sure. Changed.